docs/creating.md

```diff
@@ -84,7 +84,7 @@ Copier includes:
         context.
 
 -   `_copier_answers` includes the current answers dict, but slightly modified to make
-    it suitable to [autoupdate your project safely](configuring.md#the-answers-file):
```

Comment:
Just FYI: This anchor was invalid, but the page link method didn't raise a warning whereas the autorefs method did. :+1: